strings-csharp/HexToByteArray/Tests/Test.cs

```diff
@@ -0,0 +1,88 @@
+using HexToByteArray;
+
+namespace Tests
+{
+    public class Test
+    {
+        private readonly ConvertHex _convertHex;
+        private const string HexString = "FF008000";
+        public Test()
+        {
+            _convertHex = new ConvertHex();
+        }
+
+        [Fact]
+        public void WhenValidHexPassed_ThenConverionSuccessful()
+        {
+            byte[] expectedResult = { 255, 0, 128, 0 };
```

Comment:
Since you are reusing the same expected result along with a const string, it would be better to make this a member of the class that is defined alongside the const HexString